pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -124,11 +124,9 @@ stdenv.mkDerivation (finalAttrs: {
     # Cocoa clipboard support only works on macOS 10.14+
     ./revert-ui-cocoa-add-clipboard-support.patch
     # Standard about panel requires AppKit and macOS 10.13+
-    (fetchpatch {
-      url = "https://gitlab.com/qemu-project/qemu/-/commit/99eb313ddbbcf73c1adcdadceba1423b691c6d05.diff";
-      sha256 = "sha256-gTRf9XENAfbFB3asYCXnw4OV4Af6VE1W56K2xpYDhgM=";
-      revert = true;
-    })
+    ./revert-ui-cocoa-use-the-standard-about-panel.patch
+    # Safe area insets require macOS 11+
+    ./revert-ui-cocoa-use-safe-area-insets.patch
```

Comment:
@wegank This naming makes it sound like we're reverting a whole commit, but it's actually just a small part of [`91aa508d027`](https://gitlab.com/qemu-project/qemu/-/commit/91aa508d027).  Could we name it something without "revert", maybe?

Reply:
Does replacing "revert" with "remove" work for you?

Reply:
I like remove. Sorry for the sloppy language before.